.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
```

Comment:
It doesn't do linting, it just works inside the editor make those things happen (eg. tabs = 2 spaces, or all files get a newline at the end).